examples/sandbox/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
     "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
Not sure if we need these changes...was going for less ts-lint errors

Reply:
we can leave it since isolated to sandbox for now though still don't think the `typeRoots` are necessary or making any diff